app/controllers/application_controller.rb

```diff
@@ -13,6 +13,10 @@ class ApplicationController < ActionController::API
     render json: { error: exception.message }, status: :forbidden
   end
 
+  def current_owner
```

Comment:
I'd name this `caller` or `calling_subject`. It can be used in contexts where the calling user isn't necessarily an owner.